ketchup/main.py

```diff
@@ -1,43 +1,62 @@
+from dateutil import parser
 import datetime as dt
 from ketchup.luffy import LUFFY
-import ketchup.day_counter as dc
+import ketchup.watch_schedule_builder as dc
 import ketchup.fun_fact as fun_fact
 
 
 def run():
-    date_format = "%Y,%m,%d"
-
-    episodes = input(
-        "Hello! How many episodes is the show you are interested in watching? "
-    )
+    episodes = int(input(
+        "Hello! How many episodes are in the show you are interested in watching? "
```

Comment:
The user could as simple enter '0' as their amount, and do the same for their limit of how many episodes they would like to watch per day. 

Then when the 'actual_watch_days' variable equation is ran, the code returns a ZeroDivisionError.

Think about providing a response to the user if they failed to enter a number of episodes they would like to watch, or, giver them the option to exit.

Reply:
fixed